### ISSUE TITLE
Lua 5.2 Sample Refusing to Run

### ISSUE THREAD

**Issue by suicvne** [pspdev/psp-ports#25]
this originally started out as an issue regarding PPSSPP: https://github.com/hrydgard/ppsspp/issues/9808

and then, I borrowed a friends PSP 3000 with CFW to test out my game & and the sample and found the same exact behaviour as I did when I ran in the emulator. 

https://github.com/hrydgard/ppsspp/files/1092301/ppsspp-log.txt

^The full PPSSPP log is attached. i'm built about a week ago against the latest PSP SDK.

**Comment by suicvne** [comment]
dd6a6b2ac9b05a094d3393d7454425120f9890f2 
updating lua to 5.3.4 appears to help and i'm able to run the sample on real hardware now.

**Comment by suicvne** [comment]
this issue is something within the later versions of the SDK. i'm not sure why it doesn't properly build Lua 5.2 or 5.3

**Comment by tpimh** [comment]
I now tested the sample on PPSSPP with Lua 5.1 and Lua 5.2, it seems to work correctly.

**Comment by sharkwouter** [comment]
Thanks for testing, that's appreciated. I don't know if we still use this repo. We should probably archive it.

**Comment by tpimh** [comment]
Yes, it looks like it wasn't touched for a long time. Maybe add a link to psp-packages to readme before archiving.
